grammar/ja/src/rules.rs

```diff
@@ -1006,13 +1008,18 @@ pub fn rules_time(b: &mut RuleSetBuilder<Dimension>) -> RustlingResult<()> {
                           // true)
     // );
     b.rule_6("hour and minutes and seconds",
-                      integer_check_by_range!(0, 24),
-                      b.reg(r#"時"#)?,
-                      integer_check_by_range!(0, 59),
-                      b.reg(r#"分"#)?,
-                      integer_check_by_range!(0, 59),
-                      b.reg(r#"秒"#)?,
-                      |a, _, b, _, c, _| helpers::hour_minute_second(a.value().value as u32, b.value().value as u32, c.value().value as u32, true)
+        integer_check_by_range!(0, 24),
```

Comment:
It should be integer_check_by_range!(0, 23) as there can't be minutes after 24h in Japanese

Reply:
yes!